server/test/uk/gov/ons/addressIndex/server/controllers/AddressControllerSpec.scala

```diff
@@ -588,7 +590,7 @@ class AddressControllerSpec extends PlaySpec with Results {
         apiVersion = apiVersionExpected,
         dataVersion = dataVersionExpected,
         response = AddressByMultiUprnResponse(
-          addresses = Seq(AddressResponseAddress.fromHybridAddress(validHybridAddress, verbose = false)),
+          addresses = Seq(AddressResponseAddress.fromHybridAddress(validHybridAddress, verbose = false, pafdefault = false)),
           historical = true,
           verbose = false,
           epoch = ""
```

Comment:
Is it not necessary to have `pafdefault = false` here? I don't know but it's in others?